python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -94,8 +94,18 @@ def make_search_policies(
             raise ValueError("Invalid search policy: " + search_policy)
 
         if policy_type == "sketch":
+            if load_log_file:
```

Comment:
Do we need to check if the log file exists?

Reply:
No. This function correctly handles this case and prints
```
SearchPolicy: No measurement records found in XXX.json
```